tests/test_basin_mask.py

```diff
@@ -113,6 +113,7 @@ def test_check_size(caplog):
         nodata=-1,
         name="test",
         crs=4326,
+        lazy=True,  # create lazy dask array instead of numpy array
```

Comment:
@Tjalling-dejong Made a small change here to create a lazy array (i.e. not allocate the whole array in memory)